tools/SetupFlow/DevHome.SetupFlow/ViewModels/DevDriveViewModel.cs

```diff
@@ -260,6 +261,16 @@ public async Task ChooseFolderLocationAsync()
             if (!string.IsNullOrWhiteSpace(location?.Path))
             {
                 _log.Information($"Selected Dev Drive location: {location.Path}");
+
+                // If the user removed the file and wants to clone to the same location
+                // change Location to empty to force setting Location to location.path and
+                // enabling DevDrive validation.
+                if (FileNameAndSizeErrorList.Contains(DevDriveValidationResult.FileNameAlreadyExists)

```

Comment:
>FileNameAlreadyExists [](http://example.com/codeflow?start=79&length=21)

Why is this error special?  What if they had no disk space and then they freed some up?

'DevDriveValidationResult.NotEnoughFreeSpace'

Reply:
Correct.  The check is removed.